app/controllers/keywords_controller.rb

```diff
@@ -6,7 +6,7 @@ class KeywordsController < ApplicationController
   include Pagy::Backend
 
   def index
-    pagy, keywords = pagy(current_user.keywords)
+    pagy, keywords = pagy(current_user.keywords.order('created_at DESC').all)
```

Comment:
I think we dont need `.all` method
```suggestion
    pagy, keywords = pagy(current_user.keywords.order('created_at DESC'))
```

Reply:
Good catch

Reply:
Fixed in [`f84cf8d` (#42)](https://github.com/suho/google-search-ruby/pull/42/commits/f84cf8dbc89dd5a6e5ac9bb1503f04ae0923f7c4)